src/main/antlr4/FIRRTL.g4

```diff
@@ -94,8 +94,6 @@ stmt
   : 'wire' id ':' type info?
   | 'reg' id ':' type exp ('with' ':' reset_block)? info?
   | 'mem' id ':' info? INDENT memField* DEDENT
-  | 'cmem' id ':' type info?
-  | 'smem' id ':' type ruw? info?
```

Comment:
I thought we were going to keep around the CHIRRTL syntax for backwards compatibility?